torch/_decomp/decompositions.py

```diff
@@ -1830,31 +1838,31 @@ def upsample_bilinear2d_vec(
     n_batch, n_channels, in_h, in_w = input.shape
 
     if output_size is not None:
-        out_h = float(output_size[0])
-        out_w = float(output_size[1])
+        out_h = sym_float(output_size[0])
+        out_w = sym_float(output_size[1])
```

Comment:
test

Reply:
opinfo has nn.function.upsample_bilinear2d, so this is tested with real int. 
Do you mean we need additional test case for SymInt output_size.

Reply:
No, just that I didn't see an xfail removal for this test. So maybe we do need more inputs

Reply:
cc @albanD who did something similar for Sizes

Reply:
This was done in https://github.com/pytorch/pytorch/pull/87304